<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Conceptual Modeling for Life Sciences</title>

    <!-- Bootstrap core CSS -->
    <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="css/scrolling-nav.css" rel="stylesheet">

</head>

<body id="page-top">

<!-- Navigation -->
<nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top" id="mainNav">
    <div class="container">
        <a class="navbar-brand js-scroll-trigger" href="#page-top">CMLS@ER2022</a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive"
                aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse" id="navbarResponsive">
            <ul class="navbar-nav ml-auto">
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#about">About</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#topics">Topics of Interest</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#submission">Submission</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#special_issue">Special Issue</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#dates">Important Dates</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#organizers">Organizers</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#committee">Committee</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#program">Program</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#collaborations">Collaborations</a>
                </li>
            </ul>
        </div>
    </div>
</nav>

<header class="text-black-50" style="background-image: url('imgs/scenario_bg.jpg'); min-height: 27rem;">
    <div class="container text-center">
        <h1>The 3<sup>rd</sup> International Workshop <br> on Conceptual Modeling for Life Sciences</h1>

        <img src="imgs/logo2022.png" alt="Logo CMLS" class="logo-cmls" width="320">

        <br><br>
        <h3>17-20 October, 2022 in Hyderabad, India</h3>
        <h4 style="color:red;opacity: 0.7;">Hybrid workshop</h4>
        <!--<h3 style="color:red">Held virtually</h3>-->

        <br>
        <h4>In conjunction with the 41<sup>th</sup> International Conference on Conceptual Modeling (ER&nbsp;2022)</h4>
    </div>
</header>

<!--<section id="latest">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Latest news</h2>
                <p class="text-justify lead">We are giving a <b>10-days extension</b> for the last ones! New hard deadline is
                    <b>August 7th 2020</b>.</p>
                <p class="text-justify lead"> Important info:
                <ul class="lead">
                    <li>All accepted papers will be published by Springer in the LNCS series in the usual book that
                        collects all ER Workshop contributions
                    </li>
                    <li>We invite all papers accepted to the workshop to submit a revised and extended version to the
                        <a href="https://bmcbioinformatics.biomedcentral.com">BMC Bioinformatics journal</a> for a post-conference
                        supplement related to Conceptual Modeling in Life Sciences.
                    </li>
                <li>For the <b>best paper of the workshop</b> (as assessed by the evaluations of the members of the Program
                        Committee) the <b>Article Processing Charges of the BMC Bioinformatics supplement will be fully
                        covered</b> by the organizers (GeCo at Politecnico di Milano & PROS at Universitat Politècnica de València)
                    </li>
                </ul>
                </p>
                <p class="text-justify lead">
                    The 1st International Workshop on Conceptual Modeling for Life Sciences is going virtual, together
                    with the main conference <a href="https://er2020.big.tuwien.ac.at/"> ER 2020</a>.
                    We will soon provide further details on the platforms we will use and the
                    necessary logistics.
                    An important note upfront – there will be free registration for non-authors, and a heavily reduced
                    registration fee for authors presenting a paper. Registration with further details will be open on
                    August 15, 2020.</p>
            </div>

        </div>
    </div>
</section>-->

<!--<section id="latest">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Latest news</h2>
                <p class="text-justify lead">Thank you for everyone who joined the workshop and to all presenters for sharing their interesting research!</p>
                <p class="text-justify lead"> We made a compilation of the workshop presentations. You can watch it <a href="https://www.youtube.com/watch?v=-l4OUdrkVC4&list=PLfWxoOMC6swI59BOYNw0U3L2VNJ_dqSBG&ab_channel=bioinformaticspolimi">here on YouTube</a>.
                <p class="text-justify lead">Check also the Program (at the end of the page) for slides and videos of each video (we will keep loading them as we get permission from the authors).</p>

            </div>

        </div>
    </div>
</section>-->

<!--<section id="latest">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Latest news</h2>
                <p class="text-justify lead"><b>November 4th, 2021</b>: our BMC Bioinformatics Supplement on CMLS 2020 (Part 1) is finally out!
    Check it out here <a
                        href="https://bmcbioinformatics.biomedcentral.com/articles/supplements/volume-22-supplement-13">
                    https://bmcbioinformatics.biomedcentral.com/articles/supplements/volume-22-supplement-13</a>.
                Keep connected to know when also Part 2 is out. Also for CMLS 2021 we are planning a supplement on
                    the same Journal. The deadline for extended articles (invited) is Jan. 31st, 2022.</p>
                 <p class="text-justify lead"><b>October 19th, 2021</b>: Thank you for everyone who joined the workshop and to all presenters for sharing their interesting research!</p>

            </div>

        </div>
    </div>
</section>-->


<section id="about">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>About</h2>
                <p class="text-justify lead">
                    The recent advances in unraveling the secrets of human conditions and diseases have encouraged new
                    paradigms for their prevention, diagnosis, and treatment. As the information is increasing at an
                    unprecedent rate, it directly impacts the design and future development of information and data
                    management pipelines; thus, new ways of processing data, information, and knowledge in health care
                    environments are strongly needed.<br>
                    The third edition of the workshop aims to continue being a meeting point for Information Systems
                    (IS), Conceptual Modeling (CM), and Data Management (DM) researchers working on health care and life
                    science problems. It is also an opportunity to share, discuss and find new approaches to improve
                    promising fields, with a special focus on Genomic Data Management - how to use the information from
                    the genome to better understand biological and clinical features - and Precision Medicine - giving
                    to each patient an individualized treatment by understanding the peculiar aspects of the
                    disease.<br>
                    From the precise ontological characterization of the components involved in complex biological
                    systems to the modeling of the operational processes and decision support methods used in the
                    diagnosis and prevention of diseases, the joined research communities of IS, CM, and DM have an
                    important role to play; they must help in providing feasible solutions for a high-quality and
                    efficient health care.<br>
                    The COVID-19 pandemic has attracted increasing attention to the genetic mechanisms of humans and
                    viruses. CMLS may become an additional forum for discussing the responsibility of the conceptual
                    modeling community in supporting the life sciences related to this new reality.
                    This year – more than in the previous editions – we aim to welcome topics that have become of
                    particular interest for the general ER community: conceptual modeling for big data analytics and
                    AI-driven systems, particularly beneficial to life sciences disciplines.
                </p>
            </div>
        </div>
    </div>
</section>

<section id="topics">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Topics of interest</h2>
                <p class="text-justify lead">
                    The third edition of the workshop focuses on Conceptual Modeling as a means for facing the
                    challenges that emerge when designing and developing systems for life sciences, focused on genomics
                    and precision medicine. The workshop is not restricted to specific research methods; we will
                    consider both conceptual and empirical research, as well as novel applications.

                </p>
                <p class="text-justify lead">The topics of interest include, but are not limited to:</p>
                <ul class="lead">
                    <li>Conceptual modeling for genomics</li>
                    <li>Modeling of complex biological systems and of health ecosystems</li>
                    <li>Information systems for healthcare, genomics, or medicine of precision</li>
                    <li>Design, implementation, and evaluation of health information systems</li>
                    <li>Electronic/digital health information systems</li>
                    <li>Life science-related domain specific modeling languages</li>
                    <li>Data management and integration for genomics and biology</li>
                    <li>Ontologies and workflows for life sciences</li>
                    <li>Clinical and biological data interoperability</li>
                    <li>Interoperability of health information systems</li>
                    <li>Knowledge-representation for genetics</li>
                    <li>Business process modeling for genetic/clinical diagnosis</li>
                    <li>Conceptual model-driven big data analytics for genomics, clinical diagnosis or biological
                        problems
                    </li>
                    <li>Conceptual models for data-driven AI systems in life sciences</li>
                    <li>Models for digital transformation of healthcare systems</li>

                </ul>
                <br>
                <p class="text-justify lead">As we wish to stimulate more discussion in the ER community regarding the
                    use of models for life sciences, we welcome also “discussion papers”, particularly related to the
                    following topics:</p>
                <ul class="lead">
                    <li>Conceptual models in life sciences: from theory to practice</li>
                    <li>Models to facilitate multidisciplinary exchange in healthcare contexts</li>
                </ul>

            </div>
        </div>
    </div>
</section>

<!--<section id="accepted">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Accepted papers</h2>
                <p class="text-justify lead">
                    This is the list of accepted papers for CMLS 2020:</p>
                <ul class="lead">
                    <li>Mireia Costa, Ana León, and Óscar Pastor. <i>The Importance of the Temporal Dimension in
                        Identifying Relevant Genomic Variants: a Case Study</i></li>
                    <li>Alberto García and Juan C. Casamayor. <i>Towards the Generation of a Species-Independent
                        Conceptual Schema of the Genome</i></li>
                    <li>Mohammed R. Elkobaisi, Heinrich C. Mayr, and Vladimir A. Shekhovtsov. <i>Conceptual Human
                        Emotion Modeling (HEM)</i></li>
                    <li>Pietro Crovari, Sara Pidò, and Franca Garzotto. <i>Towards an Ontology for Bioinformatics
                        Research Process</i></li>
                    <li>Giuseppe Agapito and Mario Cannataro. <i>Using BioPAX-Parser (BiP) to annotate lists of
                        biological entities with pathway data</i></li>
                    <li>Cristian Tristão and Antonio Basilio de Miranda, Edward Hermann Haeusler, Sergio Lifschitz. <i>Relational
                        Text-type for Biological Sequences</i></li>
                </ul>
            </div>
        </div>
    </div>
</section>-->

<section id="submission">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Paper submission guidelines</h2>
                <p class="text-justify lead">
                    We invite submissions of high quality papers describing original and unpublished results
                    regarding
                    any of the workshop’s topics of interest.
                    <br><br>
                    CMLS&nbsp;2022 proceedings will be part of the <a
                        href="https://er2022web.github.io/ER2022/index.html">ER&nbsp;2022</a>
                    Workshop volume published by Springer in the LNCS series.
                    The authors must submit manuscripts using the Springer-Verlag LNCS style for Lecture Notes in
                    Computer Science.
                    For style files and details, see the page <a
                        href="http://www.springer.de/comp/lncs/authors.html">http://www.springer.de/comp/lncs/authors.html</a>.
                    The page limit for workshop papers is 10 pages.
                    Papers must be submitted as PDF files using EasyChair at <a
                        href="https://easychair.org/conferences/?conf=cmls2022" target="_blank">https://easychair.org/conferences/?conf=cmls2022</a>.
                    <br><br>
                    To ensure high quality, all papers will be thoroughly peer reviewed by the Program Committee.
                    Manuscripts not submitted in the LNCS style or having more than 10 pages will not be reviewed
                    and
                    thus automatically rejected.
                    The papers need to be original and not submitted or accepted for publication in any other
                    workshop,
                    conference, or journal.
                    Submission to CMLS 2022 will be electronically only.
                </p>
            </div>
        </div>
    </div>
</section>

<!--<section id="cameraready">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <strike><h2>Camera-ready guidelines</h2></strike>
                <strike><p class="text-justify lead">
                    For the preparation of their manuscript camera-ready version, authors should consult <a
                        href="ftp://ftp.springernature.com/cs-proceeding/svproc/guidelines/Springer_Guidelines_for_Authors_of_Proceedings.pdf">Springer’s
                    authors’ guidelines</a>
                    and use their <a
                        href="https://www.springer.com/de/it-informatik/lncs/conference-proceedings-guidelines">proceedings
                    templates</a>, either for LaTeX or for Word,
                    for the preparation of their papers. The page limit for workshop papers is 10 pages, as for the
                    previously submitted version.
                    Springer encourages authors to include their ORCIDs in their papers.
                    <br><br>
                    In addition, the corresponding author of each paper, acting on behalf of all of the authors of that
                    paper,
                    must complete and sign a <a
                        href="https://drive.google.com/file/d/1FL5_jFrqayqbPRZ2nTuNKaY55_8L8xD8/view?usp=sharing">Consent-to-Publish
                    form</a>.
                    The corresponding author signing the copyright form
                    should match the corresponding author marked on the paper.
                    Once the files have been sent to Springer, changes relating to the authorship of the papers cannot
                    be made.
                </p></strike>
            </div>
        </div>
    </div>
</section>
-->


<section id="special_issue">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Post-conference publication</h2>
                <p class="text-justify lead">
                    <!--The papers accepted to the workshop's last edition CMLS 2022,
                    published within the Springer volume <a
                        href="https://link.springer.com/book/10.1007/978-3-030-65847-2">Advances in Conceptual
                    Modeling</a>,
                    have been invited to submit a revised and extended version
                    for a post-conference supplement in the journal
                    <a
                            href="https://bmcbioinformatics.biomedcentral.com/" target="_blank"><b>BMC
                        Bioinformatics</b></a>
                    journal (2-year IF 2.511, SJR 1.374).
                    We are working towards an agreement with the same journal also for this year.-->
                    We are working towards a preliminary agreement with <a
                        href="https://bmcbioinformatics.biomedcentral.com/" target="_blank"><b>BMC
                    Bioinformatics</b></a> journal (2-year Impact Factor: 3.169)
                    for a <b>post-conference supplement</b> related to Conceptual Modeling in Life Sciences.
                    If the agreement is reached, all the papers accepted to our workshop will be invited
                    to submit a revised and extended version to the journal supplement.
                </p>
                <!--<p class="text-justify lead">For the <b>best paper</b> of the workshop (as assessed by the evaluations
                    of the members of the Program Committee) the <b>Article Processing Charges of the BMC Bioinformatics
                        supplement will be fully covered</b> by the organizers (GeCo at Politecnico di Milano & PROS at
                    Universitat Politècnica de València).-->
                </p>
            </div>
        </div>
    </div>
</section>


<section id="dates">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Important dates</h2>
                <p class="lead">
                <ul class="lead">
                    <li>Paper submission: June 15th, 2022</li>
                    <li>Notification: July 14th, 2022</li>
                    <li>Camera-ready version: July 31st, 2022</li>
                    <li>ER Conference dates: October 17-20, 2022; <h5 style="color:red">(Virtual participation/presentation is allowed in CMLS)</h5></li>
                    <!-- <li>Extended article submission to BMC Bioinformatics supplements: January 31st, 2022</li>-->

                </ul>
                </p>
            </div>
        </div>
    </div>
</section>

<!--<section id="registration">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Registration</h2>
                <p class="lead">
                    The registration for the workshop is open via the <a
                        href="https://er2020.big.tuwien.ac.at/registration/">ER website</a>.
                    At least one of the authors for each accepted paper must have an "Author Registration" (registration
                    fee of € 150,-.)
                    and attend the online conference for paper presentation.
                </p>
            </div>
        </div>
    </div>
</section>-->

<section id="organizers">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Organizers</h2>
                <p class="lead">
                    <b>Anna Bernasconi</b>, Dipartimento di Elettronica, Informazione e Bioingegneria (DEIB)
                    <br>Politecnico di Milano, Italy
                    <br>anna.bernasconi@polimi.it
                </p>
                <p class="text-justify">
                    Anna Bernasconi works as a researcher in Politecnico di Milano, within the “Data-driven Genomic
                    Computing” ERC Awarded project (2016-2021), under the supervision of Professor Stefano Ceri.
                    In 2015 she obtained a Master of Science in Computer Engineering from Politecnico di Milano and a
                    Master of Science in Computer Science from University of Illinois at Chicago with a thesis on Formal
                    Methods.
                    Her research is on bioinformatics data and metadata integration methodologies to support complex
                    biological query answering.
                    Main expertise areas include conceptual data design, data integration, data cleaning, semantic web,
                    data analysis; she is passionate about models and methods formalization.
                <p><br><br>
                <p class="lead">
                    <b>Arif Canakoglu</b>, Dipartimento di Elettronica, Informazione e Bioingegneria (DEIB)
                    <br>Politecnico di Milano, Italy
                    <br>arif.canakoglu@polimi.it
                </p>
                <p class="text-justify">
                    Arif Canakoglu currently works as a postdoctoral researcher at Politecnico di Milano;
                    he is involved the “Data-driven Genomic Computing” ERC Awarded project (2016-2021),
                    where he contributes for developing integration methods for heterogenous genomic data and
                    computational methods for genomic applications.
                    In 2016 he received his PHD on biomolecular knowledge data integration (by using a modular schema
                    data warehouse).
                    His research interests include data integration and data driven genomic computing, big data analysis
                    and processing on cloud computing,
                    as well as artificial intelligence applications. His main areas of expertise are heterogenous data
                    integration,
                    data driven/machine learning knowledge discovery approaches in genomics, and big data processes with
                    focus on cloud computing.
                </p><br><br>
                <p class="lead">
                    <b>Ana León Palacio</b>, Research Center on Software Production Methods (PROS)
                    <br>Universitat Politècnica de València, Spain
                    <br>aleon@pros.upv.es
                </p>
                <p class="text-justify">
                    Ana León, PhD in Computer Science (2019, Universitat Politècnica de València),
                    is also University Expert in Medical Genetics and Genomics by the Universidad Católica de Murcia.
                    Her main research topics include Conceptual Modeling, Genomic Data Science, Explainable AI, Data
                    Quality and Information Systems.
                    Currently, she is researcher at the Research Center on Software Production Methods (PROS-UPV)
                    where her research activity is focused on the use of conceptual models for the development of
                    Genomic Information Systems,
                    as well as the definition of a systematic process for the search, identification, load and
                    exploitation of DNA variants in the context of Precision Medicine.
                </p><br><br>
                <p class="lead">
                    <b>José Fabián Reyes Román</b>, Research Center on Software Production Methods (PROS)
                    <br>Universitat Politècnica de València, Spain
                    <br>jreyes@pros.upv.es
                </p>
                <p class="text-justify">
                    José F. Reyes R. is a researcher at PROS Research Center at Universitat Politècnica de València
                    (Spain).
                    He holds a Ph.D. in Computer Sciences (2018) from Universitat Politècnica de València (UPV, Spain),
                    a MSc in Software Engineering, Formal Methods and Information Systems (2013) from UPV (Spain),
                    a Diplomate of Analysts and Systems Designers (2011) and a University Degree in System Engineering
                    (2010)
                    from Universidad Central del Este (Dominican Republic).
                    Currently, his main research activities are centered on the use of Conceptual Models for the
                    development of Genomic Information Systems (GeIS).
                    His main research interests include Conceptual Modeling, Genomic Data Science, Engineering
                    Requirements, SE and Information Systems.
                </p>
            </div>
        </div>
    </div>
</section>

<section id="committee">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Program Committee</h2>
                <p class="lead">
                    TBD
                    <!--<ul class="lead">
                        <li>Giuseppe Agapito, Magna Graecia University</li>
                        <li>Raffaele Calogero, University of Turin</li>
                        <li>Mario Cannataro, Magna Graecia University</li>
                        <li>Matteo Chiara, University of Milan</li>
                        <li>Johann Eder, University of Klagenfurt</li>
                        <li>Jose Luis Garrido, University of Granada</li>
                        <li>Giancarlo Guizzardi, Free University of Bozen-Bolzano</li>
                        <li>Carlos Íñiguez-Jarrín, National Polytechnic School</li>
                        <li>Sergio Lifschitz, Pontifical Catholic University of Rio de Janeiro
                        </li>
                        <li>Paolo Missier, Newcastle University</li>
                        <li>Ignacio Panach, University of Valencia</li>
                        <li>Barbara Pernici, Polytechnic University of Milan</li>
                        <li>Rosario Michael Piro, Polytechnic University of Milan</li>
                        <li>Sudha Ram, University of Arizona</li>
                        <li>Maria Rodriguez Martinez, IBM Zurich Research Laboratory</li>
                         <li>Pietro Pinoli, Politecnico di Milano</li>

                    </ul>-->
                </p>
            </div>
        </div>
    </div>
</section>

<!--<section id="invited">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Invited Talk</h2>
                    <p class="text-justify">
                    We are glad to announce the CMLS 2020 Keynote Talk by <a
                        href="https://www.ncl.ac.uk/computing/people/profile/paolomissier.html#background">
                    Paolo Missier</a>: "<b>Optimising the re-execution of analytics pipelines in response to
                    changes in the data: current results, open problems, and opportunities</b>".
                    <br><br>
                    <a
                            href="https://www.ncl.ac.uk/computing/people/profile/paolomissier.html#background">
                        Paolo Missier</a> is Professor of Big Data Analytics
                    with the School of Computing at Newcastle University,
                    with 20 years experience in CS research, development, and research management.
                    <br>
                    The broad goal of his research is to understand the role of metadata, most notably data provenance,
                    in making sense of the underlying (big) data as well as improving and optimising the processes that
                    produce and extract added value from the data (i.e. through "big data" analytics)&mdash;he calls
                    this
                    metadata analytics.
                    <br>
                    He has been leading (as Principal Investigator) the ReComp project (2016-2019, EPSRC) focused on
                    preserving value from large-scale data analytics over time through selective re-computation
                    (<a href="http://recomp.org.uk/">http://recomp.org.uk/</a>)
                    where the challenge of collecting provenance metadata and extracting value from it through
                    analytics techniques is central to the research.
                    <br>

                    His talk will be focused on &mdash; but not limited to &mdash; the results of ReComp project and
                    explain
                    its application to Genomics and to the Simple Variant Interpretation (SVI) workflow.
                </p>
            </div>
        </div>
    </div>
</section>
-->

<section id="program">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Program</h2>
                <p class="lead">
                    TBD
                    <!--<h4>CMLS session is on Tuesday October 19th, 17:30-19:00 (CEST)</h4>
                    17:30-17:35
                    <br>Welcome and introduction to the workshop
                    <br>
                    <br>17:35-17:50
                    <br>Roman Lukyanenko, Veda Storey and Oscar Pastor Lopez:
                    <br>The Notion of “System” as a Core Conceptual Modeling Construct for Life Sciences.
                    <a target="_blank" class="pdf" href="presentations/1-TristaoEtAl.pdf">Slides</a>

                    <a target="_blank" class="video" href="https://youtu.be/Y74YXHj8-mE">Video</a>
                    <br>
                    </p>
                    <br>17.50-18.05
                    <br>Alberto García S., Mireia Costa Sánchez and Oscar Pastor Lopez:
                    <br>Characterization and Treatment of the Temporal Dimension of Genomic Variations: a Conceptual
                    Model-based approach.
                    <br>
                    <br>18.05-18.20
                    <br>Federico Comolli:
                    <br>Extension of the Genomic Conceptual Model to integrate Genome-Wide Association Studies.
                    <br>
                    <br>18.20-18.35
                    <br>Sara Balderas-Díaz, María José Rodríguez Fórtiz, Gabriel Guerrero-Contreras, Mercedes
                    Bellido-González and Jose Luis Garrido:
                    <br>Design of an Adaptable mHealth System Supporting a Psycho-Educational Program for Pregnant Women
                    with SGA Foetuses.
                    <br>
                    <br>18.35-19.00
                    <br>Oscar Pastor (<b>Keynote talk</b>):
                    <br>New directions for clinical and genomic problems in different diseases: A Conceptual Modeling-based
                    Approach-->
                </p>
            </div>
        </div>
    </div>
</section>

<!--<section id="accepted">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Accepted papers</h2>
                <p class="text-justify lead">
                    This is the list of accepted papers for CMLS 2021:</p>
                <ul class="lead">
                    <li>Roman Lukyanenko, Veda Storey and Oscar Pastor Lopez. <i>The Notion of “System” as a Core Conceptual Modeling Construct for Life Sciences</i></li>

                    <li>Alberto García S., Mireia Costa Sánchez and Oscar Pastor Lopez. <i>Characterization and Treatment of the Temporal Dimension of Genomic Variations: a Conceptual Model-based approach</i></li>

                    <li>Federico Comolli. <i>Extension of the Genomic Conceptual Model to integrate Genome-Wide Association Studies</i></li>

                    <li>Sara Balderas-Díaz, Rodríguez Fórtiz María José, Gabriel Guerrero-Contreras, Mercedes Bellido-González and Jose Luis Garrido. <i>Design of an Adaptable mHealth System Supporting a Psycho-Educational Program for Pregnant Women with SGA Foetuses</i></li>
                </ul>
            </div>
        </div>
    </div>
</section>-->

<section id="collaborations">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Collaborations</h2>
                This workshop is supported by the data-driven Genomic Computing group at Politecnico di Milano
                and by the VRAIN Research Center at Universitat Politecnica de Valencia (INNEST/2021/57 - Agència
                Valenciana de la Innovació and PDC2021-121243-I00 - Spanish State Research Agency)
                <p class="lead">
                    <a href="http://polimi.it/"><img src="imgs/polimi_white.jpg" alt="Logo CMLS" width="240"></a>
                    <a href="http://www.bioinformatics.deib.polimi.it/geco/" target="_blank"><img
                            src="imgs/geco_name.png"
                            alt="Logo GeCo"
                            width="170"
                            class="mr-4"></a>
                    <a href="http://www.upv.es/"><img src="imgs/upv_white.png" alt="Logo UPV" width="140" class="mr-4"></a>
                    <a href="http://www.pros.webs.upv.es/"><img src="imgs/pros.png" alt="Logo PROS" width="220"></a>
                </p>
            </div>
        </div>
    </div>
</section>


<!-- Footer -->
<footer class="py-5 bg-dark">
    <div class="container">
        <!--<p class="m-0 text-center text-white">Copyright &copy; Your Website 2019</p>-->
        <h1 class='text-light bg-dark'>Workshop on Conceptual Modeling for Life Sciences</h1>


    </div>
    <!-- /.container -->
</footer>

<!-- Bootstrap core JavaScript -->
<script src="vendor/jquery/jquery.min.js"></script>
<script src="vendor/bootstrap/js/bootstrap.bundle.min.js"></script>

<!-- Plugin JavaScript -->
<script src="vendor/jquery-easing/jquery.easing.min.js"></script>

<!-- Custom JavaScript for this theme -->
<script src="../../../Downloads/startbootstrap-scrolling-nav-gh-pages/js/scrolling-nav.js"></script>

</body>

</html>
